mod/photos.php

```diff
@@ -937,11 +937,17 @@ function photos_content(App $a)
 				$nxt = null;
 				foreach ($prvnxt as $z => $entry) {
 					if ($entry['resource-id'] == $ph[0]['resource-id']) {
-						$prv = $z - 1;
-						$nxt = $z + 1;
+						$prv = ($order_field === 'created') ? ($z - 1) : ($z + 1);
+						$nxt = ($order_field === 'created') ? ($z + 1) : ($z - 1);
```

Comment:
Parentheses are fine but superfluous in this particular case. Parentheses are necessary with ternary calls when you have several chained like this : `foo ? bar : baz ? oof : rab`. In this case the outcome is ambiguous and parentheses should be added.
```suggestion
						$prv = $order_field === 'created' ? $z - 1 : $z + 1;
						$nxt = $order_field === 'created' ? $z + 1 : $z - 1;
```

Reply:
ok